spec/lib/account_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
```

Comment:
Due to Rspec magic (see the first paragraph of your `spec_helper.rb` file), you shouldn't have to include this `require 'spec_helper'` line. Doesn't hurt, but just FYI :)